source/changelog.txt

```diff
@@ -12,6 +12,23 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.5
+------
+
+*Released April 22, 2024*
+
+Performance improvements:
+
+- :issue:`MONGOSH-1759` – Updates the stance on the trade-off between 
```

Comment:
Consider rewrite to not use "stance" and "trade-off"

Reply:
updated.